src/udev/scsi_id/scsi_id.c

```diff
@@ -373,7 +395,20 @@ static int set_inq_values(struct scsi_id_device *dev_scsi, const char *path) {
 
         dev_scsi->use_sg = sg_version;
 
-        retval = scsi_std_inquiry(dev_scsi, path);
+        switch(scsi_interface) {
```

Comment:
Please insert a space between `switch` and `(`

Reply:
Done